projects/api/__tests__/specs/auth/create.ts

```diff
@@ -10,37 +10,46 @@ interface IUserToken {
 
 describe('POST /auth/create', () => {
     it('Creates a user', async () => {
+        // Creating the data to send to create the user
         const user: IStringTMap<string> = {
             firstName: 'John',
             lastName: 'Doe',
             email: 'john.doe@bitsky.be',
             password: 'iliketrains!',
         }
```

Comment:
Maybe we can create like a __mockups__ folder with all the mockups and for example here the John Doe user we use several times in tests?